src/diffusers/schedulers/scheduling_edm_dpmsolver_multistep.py

```diff
@@ -21,10 +21,15 @@
 import torch
 
 from ..configuration_utils import ConfigMixin, register_to_config
+from ..utils.import_utils import OptionalDependencyNotAvailable, is_torchsde_available
 from ..utils.torch_utils import randn_tensor
 from .scheduling_utils import SchedulerMixin, SchedulerOutput
 
 
+if is_torchsde_available():
+    from .scheduling_dpmsolver_sde import BrownianTreeNoiseSampler
+
+
 class EDMDPMSolverMultistepScheduler(SchedulerMixin, ConfigMixin):
```

Comment:
can we make a `CosineDPMSolverMultistepScheduler` and move all the changes there? it is ok to only support BrownianTree there 

it is not EDM and the `DPMSolverMultistepScheduler` is super bloated already, I think the easiest way is to make a new one!

Reply:
Of course, I'm not sure about how to best describe it in docstrings and docs though!

Reply:
thanks! I updated it here https://github.com/huggingface/diffusers/pull/8716#discussion_r1696170031 
The scheduler itself is not something entirely new - but this combination was not used in any other models at least in diffusers I think, and the "cosine schedule" part is the only part that's not in the DPM scheduler so let's just make a simple note of that